### ISSUE TITLE
K6_SYSTEM_TAGS is returning an error when set

### ISSUE THREAD

**Issue by markjmeier** [loadimpact/k6#768]
When executing a test with the `K6_SYSTEM_TAGS` Env variable an error is returned.  e.g.:

```
export 
K6_SYSTEM_TAGS=proto,subproto,status,method,url,name,group,check,error,tls_version,vu,iter
```
returns 

```
ERRO[0001] envconfig.Process: assigning K6_SYSTEM_TAGS to SystemTags: 
converting 
'proto,subproto,status,method,url,name,group,check,error,tls_version,vu,iter' 
to type lib.TagSet. details: invalid map item: "proto"
```

The flag `--system-tags` does NOT produce this error and works as expected.

**Comment by na--** [comment]
I think this happens because `envconfig` tries to work with the underlying map that's currently used to track the enabled system tags. To fix that we probably have to implement the [`TextUnmarshaler`](https://golang.org/pkg/encoding/#TextUnmarshaler) interface for the type: https://github.com/kelseyhightower/envconfig#supported-struct-field-types

The biggest issue with fixing this is that both parts of the equation would soon change - we'd probably have to replace `envconfig` pretty soon (because of this https://github.com/loadimpact/k6/issues/671 and a few other issues),  and we also probably should change the underlying type used to track the enabled system tags (https://github.com/loadimpact/k6/issues/755)... So we'll either do a quick fix that would soon be overwritten or fix this as part of one of the aforementioned issues.

**Comment by cuonglm** [comment]
@na-- This issue seems to be fixed.

**Comment by na--** [comment]
Hmm unfortunately, the error seems to have disappeared at some point, but instead of being fixed, I think it's just not triggered, since the current k6 doesn't seem to read the `K6_SYSTEM_TAGS` variable at all. Running ` K6_SYSTEM_TAGS=proto,status k6 run -o json script.js` with k6 v0.24.0 or v0.25.0 doesn't produce the error, but it also doesn't change the used system tags, while k6 v0.23.1 produces the error reported above...

**Comment by na--** [comment]
Ah, actually, my mistake. It seems like @MStoykov partially fixed this [here](https://github.com/loadimpact/k6/commit/8bcf39ac2d5d4c4b8d42dda25dedfe5b7fe3a64f#diff-2a3d79a0a11711742c3804c5e3846597R79) in the datadog/statsd [pull request ](https://github.com/loadimpact/k6/pull/915), but it was masked because of the [CLI option handling bug](https://github.com/loadimpact/k6/issues/1010), which you [recently fixed](https://github.com/loadimpact/k6/pull/1092). So @cuonglm, you are completely right, this is now fixed - it's working perfectly on `master` and it's soon going to be released in v0.25.1.
